nixos/tests/networking.nix

```diff
@@ -72,6 +72,7 @@ let
   testCases = {
     loopback = {
       name = "Loopback";
+      machine.networking.useDHCP = false;
```

Comment:
can't we just stip all lines setting `networking.useDHCP = false;` in the tests, if we set it for each interface?

Reply:
We now assert `useDHCP` to be false in conjunction with networkd, I didn't want to change the whole test suite that for the most part doesn't use networkd.